client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -89,6 +89,17 @@ export const useCommandsArrayWpcom = ( {
 		displaySuccessNotice( __( 'Copied new password' ) );
 	};
 
+	const openPHPmyAdmin = async ( siteId: number ) => {
+		const { token } = await wpcom.req.post( {
+			path: `/sites/${ siteId }/hosting/pma/token`,
+			apiNamespace: 'wpcom/v2',
+		} );
+
+		if ( token ) {
+			window.open( `https://wordpress.com/pma-login?token=${ token }` );
+		}
+	};
```

Comment:
✨

Reply:
I am wondering if there is no token, should we redirect to Hosting Configuration similar to the cases with SSH and resetting the password?